posthog/storage/object_storage.py

```diff
@@ -121,5 +160,15 @@ def read_bytes(file_name: str) -> Optional[bytes]:
     return object_storage_client().read_bytes(bucket=settings.OBJECT_STORAGE_BUCKET, key=file_name)
 
 
+def list_objects(prefix: str) -> Optional[List[str]]:
+    return object_storage_client().list_objects(bucket=settings.OBJECT_STORAGE_BUCKET, prefix=prefix)
+
+
+def get_presigned_url(file_key: str, expiration: int = 3600) -> Optional[str]:
```

Comment:
This is a long ttl given that we expect it to be immediately loaded. Shall we either make it here or in the caller 600 (10 mins) instead?